REFERENCE.md

```diff
@@ -145,27 +146,6 @@ Apache server's or a virtual host's listening address and port.
 * `apache::peruser::processor`: Enables the `Peruser` module for FreeBSD only.
 * `apache::security::rule_link`: Links the activated_rules from `apache::mod::security` to the respective CRS rules on disk.
 
-### Resource types
```

Comment:
These shouldn't be deleted because they're still present. Any idea why it's doing this?

Reply:
Because i used puppet strings wrongly due to fixtures, which aren't ignored by it. I think i managed it now to have a correct reference